moc_openstack_api_reporting.py

```diff
@@ -0,0 +1,572 @@
+#!/usr/bin/python3
+""" This is intended to poll openstack for changes to be reported in xdmod """
+# pylint: disable=line-too-long too-many-locals
```

Comment:
I don't think you need to disable the `too-many-locals` check.

Reply:
I must have refactored the code so that this is no longer necessary.  This is the basic problem I have with disabling lint errors at the file scope, instead of, say the class or the function scope.